### DISCUSSION TITLE
Argument while running qdrant.

### DISCUSSION THREAD

**Discussion by cravishe** [qdrant/qdrant#3756] [General]
Hello,
[main.rs](https://github.com/qdrant/qdrant/blob/d0baa5ef4a4bc2c597b28d0e1f1011262be8ed4d/src/main.rs#L114-L116) of qdrant seems to expect command line arguments when running (please correct me if I am wrong). Is it mandatory? What arguments can be passed for it.

Thank you

**Comment by timvisee** [comment]
Yes, you can optionally provide arguments.

When running Qdrant in distributed mode - using multiple nodes - providing arguments is required.

See: https://qdrant.tech/documentation/guides/distributed_deployment/#distributed-deployment
